docs/configuration/environment-variables.md

```diff
@@ -35,10 +40,10 @@ When setting or unsetting environment variables, you may wish to avoid an applic
 dokku config:set --no-restart node-js-app ENV=prod
 ```
 
-If you wish to have the variables output in an `eval`-compatible form, you can use the `--export` flag:
+If you wish to have the variables output in an `eval`-compatible form, you can use the `config:export` command
```

Comment:
Does the export flag no longer exist?

Reply:
@josegonzalez yeah, good point. I saw that we are trying to move away from bare `dokku blah` commands and instead favor `dokku blah:subcommand` commands, right? Would we like to preserve the old functionality? I not what should happen on bare `dokku config`?

Reply:
Thats fine, just want to make sure that was the intention.

Reply:
There were some small issues with the docs that I just fixed. I also added `--shell` and `--export` back to `dokku config`, albeit marked as deprecated.